packages/ui/src/components/MenuV2/types.ts

```diff
@@ -63,6 +63,7 @@ export type MenuProps = {
    */
   hideOnClickItem?: boolean
   footer?: ReactNode
+  nested?: boolean
```

Comment:
Seen with you in pm, we need to avoid this prop and try to make it automatic in popup or it will cause issues when the menu is at the edge of the right of the screen